resources/assets/less/bem/profile-links.less

```diff
@@ -36,4 +42,27 @@
     font-weight: 600;
     word-wrap: break-word;
   }
+
+  .edit-profile-links {
+    position: absolute;
+    top: 0;
+    right: 0;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 70px;
```

Comment:
this should be `@gutter-profile-page` on default and `@gutter-profile-page-desktop` on desktop. Check `page-extra__actions`.